File: php/tests/test_profile_update_unauthorized.php
<?

require_once('../EpiDBClient.php');
require_once('config.php');

$key = $_key;
$data = array(
    'name' => 'John Doe',
    'address' => 'Jakarta'
);

$client = new EpiDBClient();
$client->server = $_server;
$res = $client->profile_update($_user_id, $data);

var_dump($res);

// vim: ts=4 sts=4 expandtab

File: python/tests/test_profile_update_unauthorized.py
#!/usr/bin/env python

import sys
sys.path += ['../']

from epidb.client import EpiDBClient
import config

key = config.key
data = {
    'name': 'John Doe',
    'address': 'Jakarta'
}

client = EpiDBClient()
client.server = config.server
res = client.profile_update(config.user_id, data)

print res

# vim: ts=4 sts=4 expandtab

